index.js

```diff
@@ -0,0 +1,52 @@
+function getRandomIntInclusive(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+// console.log(getRandomIntInclusive(1,6));
+
+const dice = {
+	value : 1,
+	roll: function(){
+		this.value = getRandomIntInclusive(1,6);
+		return this.value;
+	}
+};
+
+const diceTwo = {
+	value : 1,
+	roll: function(){
+		this.value = getRandomIntInclusive(1,6);
+		return this.value;
+	}
+};
+
+function printDice(diceOneNumber, diceTwoNumber) {
+	let newClass = "img-0"+diceOneNumber;
+	let elm = document.getElementById("dice-01");
+
+	if(elm.className !== newClass){
+		elm.className = newClass;
+	}
+	let newClassTwo = "img-0"+diceTwoNumber;
+	let elmTwo = document.getElementById("dice-02");
+
+	if(elmTwo.className !== newClassTwo){
+		elmTwo.className = newClassTwo;
+	}
+}
+
+
+document.querySelector("button").addEventListener("click", event => {
+	event.preventDefault();
+	let diceOneNumber = dice.roll();
+	// let containerOne = document.querySelector(".dice-01");
+
+	let diceTwoNumber = diceTwo.roll();
+	// let containerTwo = document.querySelector(".dice-02");
+	printDice(diceOneNumber, diceTwoNumber);
+
+});
+
+printDice(dice.roll(), diceTwo.roll());
```

Comment:
As mentioned above, If you actually set the initial value for the dice to be a random number, you don't actually have to roll the dice here. you can simply call `printDice()` with the value property of each dice:

```javascript
printDice(dice.value, dice2.value);
```

Reply:
I tried that, but the problem was that when the user open the page, it's didn't display the dice, and also when refresh the page too,